usb-arduino.c

```diff
@@ -177,15 +177,15 @@ static int arduino_probe(struct usb_interface * interface, const struct usb_devi
 	for (i = 0; i < arduino_currsetting->desc.bNumEndpoints; ++i) {
 		endpoint = &arduino_currsetting->endpoint[i].desc;
 		if (((endpoint->bEndpointAddress & USB_ENDPOINT_DIR_MASK) == USB_DIR_IN)
-		    && ((endpoint->bmAttributes & USB_ENDPOINT_XFERTYPE_MASK)
+			&& ((endpoint->bmAttributes & USB_ENDPOINT_XFERTYPE_MASK)
 			== USB_ENDPOINT_XFER_BULK))
 		{
 			dev->bulk_in_endpoint = endpoint;
 			printk("Found Bulk Endpoint IN\n");
 		}
 		
 		if (((endpoint->bEndpointAddress & USB_ENDPOINT_DIR_MASK) == USB_DIR_OUT)
-		    && ((endpoint->bmAttributes & USB_ENDPOINT_XFERTYPE_MASK)
+			&& ((endpoint->bmAttributes & USB_ENDPOINT_XFERTYPE_MASK)
```

Comment:
oh this looks different with tab length of 8, just change it back if you dont want this